pom.xml

```diff
@@ -1022,6 +1022,19 @@
     </pluginRepositories>
 
     <profiles>
+        <profile>
+            <!-- Adds an SLF4J implementation, useful when developing within an IDE -->
```

Comment:
Maybe we can add this as test dependency in general?
Hm... for core, we have this already.
In java, it is missing.
Or do you need it, when you run CLI from within an IDE?

I'll keep it in, but would be interested in how you use it...

Reply:
I think it was to run tests like CliTest (in module pmd-java). Intellij would run them but print a warning that there is no logger implementation, and hid the messages. I'm not sure a profile is the best way, maybe just having the test dependency as you suggest would be better...

Reply:
@adangel I remember now where I needed this. I have a bunch of run configurations for Intellij which run the main class PMD with different arguments. Most of them use the classpath of pmd-java, so that you can launch them quicker (Intellij only needs to build pmd-java and its dependencies, not all modules). But Intellij doesn't use test dependencies when a run configuration depends on a module, so without the profile there would be no slf4j impl in this setup.